pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -550,6 +552,19 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 	return reconcile.Result{RequeueAfter: ResyncPeriod}, nil
 }
 
+func updateIPsecTelemetry(newOperConfigSpec *operv1.NetworkSpec) {
```

Comment:
`updateIPsecMetric(`

Theres no telemetry update here - just updating a metric. Gathering of telemetry based on this metric is outside the scope of this PR.